src/Gui/CMakeLists.txt

```diff
@@ -66,6 +68,7 @@ endif(MSVC)
 if(MSVC)
     set(FreeCADGui_LIBS
          FreeCADApp
+         Part
```

Comment:
Adding extension modules to the core system is a no go. This breaks the architecture!

Reply:
He mentioned in the description that this is temporary.

Reply:
Yeah, I have seen it now, too.